marklogic-data-hub-central/ui/e2e/cypress/integration/scenarios.json.tsx

```diff
@@ -6,21 +6,20 @@ import detailPage from '../support/pages/detail';
 import homePage from "../support/pages/home";
 import { Application } from '../support/application.config';
 
-
 xdescribe('json scenario on view entities page', () => {
 
   //login with valid account
   beforeEach(() => {
     cy.visit('/');
     cy.contains(Application.title);
-    cy.loginAsDeveloper();
+    cy.loginAsDeveloper().withUI();
```

Comment:
Is this update required on a commented test?

Reply:
@bsrikan not required, I was testing something and added it, will remove.